airbyte-ci/connectors/pipelines/pipelines/dagger/actions/system/docker.py

```diff
@@ -46,10 +50,16 @@ def with_global_dockerd_service(dagger_client: Client) -> Container:
         .with_exposed_port(DOCKER_HOST_PORT)
         # Mount the docker cache volumes.
         .with_mounted_cache("/tmp", dagger_client.cache_volume(DOCKER_TMP_VOLUME_NAME))
```

Comment:
Would this be a good candidate for a PRIVATE cache mount? Food for thought.

Reply:
I don't think so, quite the opposite, as this volume is made to enable file sharing between the docker client and the host 😄 
Let's say we have a `docker run` execution inside the containerized docker host that writes to `/tmp` (through a volume mount): the mounted volume is the only way that the bound client can read the file